logsink/logsink.go

```diff
@@ -0,0 +1,62 @@
+package logsink
+
+import (
+	"context"
+)
+
+// Logs is a list of logs
+type Logs []*Log
+
+// Log is a record or event that has occurred
+type Log struct {
+	TraceID        string                 `json:"traceId,omitempty"`
+	SpanID         string                 `json:"spanId,omitempty"`
+	SeverityText   string                 `json:"severityText,omitempty"`
+	ShortName      string                 `json:"shortName,omitempty"`
+	Body           interface{}            `json:"body"`
+	TimeStamp      int64                  `json:"timestamp"`
+	Resource       map[string]interface{} `json:"resource,omitempty"`
+	Attributes     map[string]interface{} `json:"attributes,omitempty"`
+	TraceFlag      byte                   `json:"traceFlags,omitempty"`
+	SeverityNumber int8                   `json:"severityNumber,omitempty"`
```

Comment:
I assume these are at the end for better packing due to field alignment.

Reply:
yes